shared/folders/index.js

```diff
@@ -41,6 +41,7 @@ class Folders extends Component<void, Props, State> {
       <Render
         {...this.props.folderProps}
         onClick={path => this.props.routeAppend(path)}
+        onRekey={path => this.props.routeAppend(path)}
```

Comment:
turns out this doesn't actually need to be different in this case. leaving this in case it does change in the future and we'd have to plumb it all back through